packages/astro/test/entry-file-names.test.js

```diff
@@ -0,0 +1,26 @@
+import { expect } from 'chai';
+import * as cheerio from 'cheerio';
+import { loadFixture } from './test-utils.js';
+
+describe('vite.build.rollupOptions.entryFileNames', () => {
+	let fixture;
+
+	before(async () => {
+		fixture = await loadFixture({
+			root: './fixtures/entry-file-names',
+		});
+		await fixture.build();
+	});
+
+	it('Renders correctly', async () => {
+		const html = await fixture.readFile('/index.html');
+		const $ = cheerio.load(html);
+		expect($('#hello')).to.have.a.lengthOf(1);
+	});
+
+	it('Outputs a client module that was specified by the config', async () => {
+		const js = await fixture.readFile('/assets/js/Hello.js');
```

Comment:
Just double checking, does this impact how deployment adapters find the main server entry point or will they pick this up automatically?